pkg/operation/botanist/monitoring.go

```diff
@@ -300,6 +288,47 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 	return nil
 }
 
+func (b *Botanist) DeploySeedGrafana(ctx context.Context) error {
```

Comment:
Most of the code of this method is duplicated in `DeploySeedMonitoring`. 
Have you tried to call `DeploySeedGrafana` from `DeploySeedMonitoring` based on a function parameter, e.g. `withGrafana bool`?

Reply:
No